tests/unit/utils/test_file_utils.py

```diff
@@ -14,7 +14,6 @@ def test_load_file(self):
         file = utils.load_file(os.path.join(dirname, "../../resources/foobar-many.txt"))
 
         assert file.decode("utf-8").startswith("foobar foobar foobar")
-        assert len(file.decode("utf-8")) == 4563
```

Comment:
I think this is the only change that surprised me. Is the txt file size os dependent? Is this related to different line endings or something?

Reply:
Yeah must be from line endings -- it was slightly different per OS.